### ISSUE TITLE
cron job does not escape `%` characters

### ISSUE THREAD

**Issue by devmage** [ansible/ansible-modules-core#3216]
##### Issue Type:
- Bug Report
##### Component Name:

cron module
##### Ansible Version:

```
ansible 2.0.0.1
  config file = /home/admin/.ansible.cfg
  configured module search path = Default w/o overrides
```
##### Ansible Configuration:

```
[defaults]
inventory=~/bin/our-project-inventory
```
##### Environment:

`Linux ip-10-254-0-230 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt20-1+deb8u4 (2016-02-29) x86_64 GNU/Linux`
##### Summary:

Reopening of https://github.com/ansible/ansible/issues/14853

Cron tasks as configured by the `cron` module do not escape `%` symbols. Documentation around the `cron` module does not specify that `%` symbols, if intended as literal characters, must be escaped. This causes early termination of cron tasks, as [unescaped `%` characters are interpreted as newlines in cron](https://en.wikipedia.org/wiki/Cron#Format). This can cause unintended behavior in cron tasks.

This could reasonably be explained as a #wontfix and/or a feature of cron. However, it would be helpful then to at least document this behavior, and to document that Ansible does not attempt to escape cron syntax in `cron` entries.
##### Steps To Reproduce:

Create a `cron` task in a playbook, and run that playbook: 

```
- cron: name="delayed script" job="/bin/bash -c '/bin/sleep $(/usr/bin/expr $RANDOM % 60)m && script.sh"
```

This expression allows for a random sleep of 0-59 minutes before running `script.sh`. You will now have an entry in that machine's `/etc/cron.d/` as expected.

##### Expected Results:

The cron job sleeps for a random period, then `script.sh` runs without error.
##### Actual Results:

Per cron syntax, the `%` character above is interpreted as a newline, the expression fails, and `script.sh` is never executed. To correct this for cron, the `%` must be escaped, as `$RANDOM \% 60`.

**Comment by rsoletob** [comment]
+1

**Comment by ansibot** [comment]
@dsummersl, @EvanK, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by EvanK** [comment]
This is documented in #3770, so bug_resolved?
